src/accessors/ISettingsExtend.ts

```diff
@@ -0,0 +1,22 @@
+import { ISetting } from '../settings/index';
+
+/**
+ * This accessor provides methods for adding custom settings
+ * of Rocket.Chat in a compatible way. Use it during initialization of your Rocketlet
+ */
+
+export interface ISettingsExtend {
+
+    /**
+     * Adds a group into which settings can be added lateron
+     * @param name The technical name of the group to be referred to in the settings
+     */
+    provideGroup(name: string, label: string): void;
```

Comment:
since those methods would be executed during installation and startup, their only way of communicating errors and messages would be exceptions and `console`